composition/content_fetcher.go

```diff
@@ -15,10 +16,11 @@ func (def *FetchDefinition) IsFetchable() bool {
 }
 
 type FetchResult struct {
-	Def     *FetchDefinition
-	Err     error
-	Content Content
-	Hash    string // the hash of the FetchDefinition
+	Def        *FetchDefinition
+	Err        error
+	Content    Content
+	HttpStatus int
```

Comment:
The Http Status Code was already part of Content.httpStatusCode
so, there is no need to double it here in the fetch result.